sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -697,6 +714,35 @@ def update_port_transceiver_status_table_sw(logical_port_name, status_tbl, statu
     fvs = swsscommon.FieldValuePairs([('status', status), ('error', error_descriptions)])
     status_tbl.set(logical_port_name, fvs)
 
+def update_port_transceiver_status_table_sw_cmis_state(lport, xcvr_table_helper, port_mapping, cmis_state_to_set):
```

Comment:
@mihirpat1 any specific reason to define outside CmisManagerTask? if not, can we keep this inside CMIS task class

Reply:
I have moved this to CmisManagerTask now.